Library/Formula/jsonnet.rb

```diff
@@ -0,0 +1,44 @@
+class Jsonnet < Formula
+  desc "A domain specific configuration language for defining JSON data."
```

Comment:
Need to kill off the `A` at the start here. Just `Domain specific configuration language for defining JSON data`